WordPress/src/main/java/org/wordpress/android/ui/AddQuickPressShortcutActivity.java

```diff
@@ -113,8 +112,7 @@ public void onItemClick(AdapterView<?> arg0, View row, int position, long id) {
 
         } else {
             // no account, load new account view
-            Intent i = new Intent(AddQuickPressShortcutActivity.this, SignInActivity.class);
-            startActivityForResult(i, ADD_ACCOUNT_REQUEST);
+            ActivityLauncher.showSignInForResult(AddQuickPressShortcutActivity.this);
```

Comment:
`ActivityLauncher.showSignInForResult` uses `RequestCodes.ADD_ACCOUNT` as a request code but `AddQuickPressShortcutActivity.java` expects ADD_ACCOUNT_REQUEST in its `onActivityResult`. Will this still work?

Reply:
I don't think so.  Good catch!  I updated the request code in 46c88c1032a3eb640dddaa617cb44f6c27a09f86.